android_webview/test/shell/tango/jni/MathUtils.cpp

```diff
@@ -2,16 +2,17 @@
 
 #include "gtx/quaternion.hpp"
 #include "gtc/type_ptr.hpp"
+#include "gtc/matrix_transform.hpp"
 
 namespace tango_chromium {
 
 void matrixFrustum(float const & left,
-             float const & right,
-             float const & bottom,
-             float const & top,
-             float const & near,
-             float const & far,
-             float* matrix) {
+    float const & right,
```

Comment:
isn't it standard to not have space between const and &?

Reply:
Have to admit that copied and pasted this function signature. I will fix it.